tests/conftest.py

```diff
@@ -9,8 +10,11 @@ def fixture_dir():
 
 
 @pytest.fixture
-def api(fixture_dir):
-    return apypie.Api(uri='https://api.example.com', apidoc_cache_dir=fixture_dir.strpath, apidoc_cache_name='dummy')
+def api(fixture_dir, requests_mock):
+    with open(str(fixture_dir) + '/dummy.json', "r") as read_file:
+        data = json.load(read_file)
+    requests_mock.get('https://api.example.com/apidoc/v1.json', json=data)
```

Comment:
I think instead of using `fixture_dir` here, we should pass `apidoc_cache_base_dir=tmpdir.strpath` and let the code generate _dir and _name.

See https://docs.pytest.org/en/latest/builtin.html for details about the tmpdur fixture of pytest.